pkgs/build-support/setup-hooks/multiple-outputs.sh

```diff
@@ -73,6 +77,7 @@ _multioutConfig() {
         --docdir=${!outputDoc}/share/doc/${shareDocName} \
         --libdir=${!outputLib}/lib --libexecdir=${!outputLib}/libexec \
         --localedir=${!outputLib}/share/locale \
+        --enable-static \
```

Comment:
Certainly this should only be added if there is a `static` output. Even then, I fear it is to prescriptive but I'd love to be wrong!

Reply:
_If_ doing this, we should better also remove the `--disable-static` flag (on line ~963).